v3/cypress/support/elements/TableTile.ts

```diff
@@ -0,0 +1,126 @@
+    },
+    deleteCase() {
+        this.getIndexMenu().should("be.visible");
+        cy.clickMenuItem("Delete Case");
+        cy.wait(500);
+    },
+    getInsertCasesModalHeader() {
+        return cy.get("[data-testid=codap-modal-header]");
+    },
+    closeInsertCasesModal() {
+        cy.get("[data-testid=modal-close-button]").click();
+    },
+    cancelInsertCasesModal() {
+        cy.get("[data-testid=Cancel-button]").click();
+    },
+    getAttributeHeader() {
+        return cy.get("[data-testid^=codap-attribute-button]");
+    },
+    getAttribute(name) {
+        return cy.get("[data-testid^=\"codap-attribute-button " + name + "\"]");
+    },
+    openAttributeMenu(name) {
+        this.getAttribute(name).click();
+    },
+    getAttributeMenuItem(item) {
+        return cy.get("[data-testid=attribute-menu-list] button").contains(item);
+    },
+    selectMenuItemFromAttributeMenu(item) {
+        this.getAttributeMenuItem(item).click();
+    },
+    renameColumnName(newName) {
+        cy.get("[data-testid=column-name-input").type(newName);
+    },
+    // Edit Attribute Property Dialog
+    enterAttributeName(name) {
+        cy.get("[data-testid=attr-name-input]").type(name);
+    },
+    enterAttributeDescription(text) {
+        cy.get("[data-testid=attr-description-input]").type(text);
+    },
+    selectAttributeType(type) {
+        cy.get("[data-testid=attr-type-select]").select(type);
+    },
+    enterAttributeUnit(unit) {
+        cy.get("[data-testid=attr-unit-input]").type(unit);
+    },
+    selectAttributePrecision(number) {
+        cy.get("[data-testid=attr-precision-select]").select(number);
+    },
+    selectAttributeEditableState(state) {
+        cy.get("[data-testid=attr-editable-radio] span").contains(state).click();
+    },
+    getApplyButton() {
+        return cy.get("[data-testid=Apply-button]");
+    },
+    editAttributeProperty(attr, name, description, type, unit, precision, editable) {
+        this.openAttributeMenu(attr);
+        this.selectMenuItemFromAttributeMenu("Edit Attribute Properties...");
+        if (name!=="") {
+            this.enterAttributeName(`{selectAll}{backspace}` + name);
+        }
+        if (description!=null) {
+            this.enterAttributeDescription(`{selectAll}{backspace}` + description);
+        }
+        if (type!=null) {
+            this.selectAttributeType(type);
+        }
+        if (unit!=null) {
+            this.enterAttributeUnit(`{selectAll}{backspace}` + unit);
+        }
+        if (precision!=null) {
+            this.selectAttributePrecision(precision);
+        }
+        if (editable!=null) {
+            this.selectAttributeEditableState(editable);
+        }
+        this.getApplyButton().click();
+        
+    },
+    getCell(line, row, instance) {
+        return cy.get("[data-testid=case-table] [aria-rowindex=\"" + row + "\"] [aria-colindex=\"" + line + "\"] .cell-span");
```

Comment:
```suggestion
        return cy.get(`[data-testid=case-table] [aria-rowindex="${row}"] [aria-colindex="${line}"] .cell-span`);
```
Use template strings rather than string concatenation.

Reply:
Done